### DISCUSSION TITLE
SIP003 plugin 在Linux上为什么没有使用UNIX Domain Socket呢？

### DISCUSSION THREAD

**Discussion by 18o** [shadowsocks/shadowsocks-rust#1032] [General]
如题，这样会对性能好一些吧

**Comment by zonyitoo** [comment]
Well, it should. But it will cause inconsistency in other systems, which don't have UDS. On the other hand, there shouldn't have any noticeable performance differences between `lo` and UDS in shadowsocks' use-case.
